lib/rubocop/formatter/disabled_config_formatter.rb

```diff
@@ -23,6 +23,12 @@ class << self
         attr_accessor :config_to_allow_offenses
       end
 
+      def file_started(_file, file_info)
```

Comment:
I should note that the code I based this on just set an instance variable inside of the formatter from the cli lib directly, which I also didn't like much -- but I can totally put it back to that if you prefer.